src/main/scala/com/mwz/sonar/scala/ScalaPlugin.scala

```diff
@@ -47,6 +47,16 @@ object Scala {
   private val DefaultScalaVersion = ScalaVersions.Scala_2_11.toString()
   private val SourcesPropertyKey = "sonar.sources"
   private val DefaultSourcesFolder = "src/main/scala"
+  private val Test: Option[String] = Some("test")
+
+  //test
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Space after the start of the comment [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.sonar-scala.com/coding_rules#rule_key=Scalastyle%3Ascalastyle_SpaceAfterCommentStartChecker)